Objects/Converters/ConverterRevit/ConverterRevitShared/ConverterRevit.cs

```diff
@@ -375,6 +387,7 @@ public bool CanConvertToSpeckle(object @object)
         DB.Architecture.TopographySurface _ => true,
         DB.Wall _ => true,
         DB.Mechanical.Duct _ => true,
+        DB.Mechanical.Space _ => true,
```

Comment:
I think we're missing a `CanConvertToSpeckle` for the `SpaceSeparationLines`.

Reply:
They come through as `ModelLines`, as the `RoomBoundaryLines` do, so they are captured in the `DB.ModelCurve` case.